task-2-SERVLETS/materials/Materials.md

```diff
@@ -0,0 +1,11 @@
+# Materials
+## Links:
+### Servlets
+- [Servlet API classes](https://beginnersbook.com/2013/05/servlet-api/)
+- [Introduction to Servlets and Servlet Containers](https://www.baeldung.com/java-servlets-containers-intro)
+- [Introduction to Java Servlets](https://www.baeldung.com/intro-to-servlets)
+- [Java EE for beginners](https://youtube.com/playlist?list=PLAma_mKffTOTTFqIkLXgHqVuL6xJhb0mr)
+
+### Pattern: Command
```

Comment:
Let's also add this:
https://www.youtube.com/watch?v=T3oXyVYmkyY

Reply:
I don't think that this additional link could be useful. Although, students should be able to google it in case they would have any questions